Sources/GRPC/ClientConnection.swift

```diff
@@ -54,6 +55,8 @@ import NIOTLS
 ///
 /// See `BaseClientCall` for a description of the remainder of the client pipeline.
 public class ClientConnection {
+  internal static let logger = Logger(subsystem: .clientChannel)
```

Comment:
Why is this one static?

Reply:
It's used by static functions. Given I was going to add a connection id this will have to change anyway.